### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by pingpongboss**

### CODE REVIEW

File: 03_08_2018/bug-hunt/build.gradle [hunk 1]
```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
```
Comment (artem-zinnatullin) [inline 03_08_2018/bug-hunt/build.gradle]: maybe rename folder to have year first, then month then day so that it will be alphabetically sortable?

**Comment by pingpongboss** [comment]
Fixed folder name

**Comment by pingpongboss** [comment]
🚀

**Comment by artem-zinnatullin** [comment]
no SQ to handle your 🚀

**Comment by pingpongboss** [comment]
Ohhh my god I can merge this PR now.

**Pull request merged**